Application/EdFi.Ods.Api.Startup/Features/Installers/CoreApiInstaller.cs

```diff
@@ -235,22 +242,26 @@ private void RegisterUniqueIdToUsiValueMapper(IWindsorContainer container)
 
         private void RegisterPersonUniqueIdToUsiCache(IWindsorContainer container)
         {
-            container.Register(
-                Component
-                    .For<IPersonUniqueIdToUsiCache>()
-                    .ImplementedBy<PersonUniqueIdToUsiCache>()
-                    .DependsOn(Dependency.OnValue("synchronousInitialization", false)));
+            container.Register(Component.For<IPersonUniqueIdToUsiCache>()
```

Comment:
The original ticket was specifically about people not being able to recreate after delete, so shouldn't this cache also be consuming the ExpiringConcurrentDictionaryCacheProvider?

Reply:
As per standup, this is still using the Asp.Net cache, so it correctly handles expirations.